arch/arm/src/rp2040/Kconfig

```diff
@@ -663,6 +663,211 @@ endif # ADC
 
 endif # RP2040_ADC
 
+#####################################################################
+#  CLK_GPOUT Configuration
+#####################################################################
+menu "GPIO Clock Output"
+config RP2040_CLK_GPOUT_ENABLE
+	bool "Enable Clock Outputs"
+	default n
+
+config RP2040_CLK_GPOUT0
+	bool "CLK_GPOUT0 Clock Output (pin 21)"
+	depends on RP2040_CLK_GPOUT_ENABLE
+	default n
+	---help---
+		Enable CLK_GPOUT0
+
+if RP2040_CLK_GPOUT0
+choice
+	prompt "Source Clock"
+	depends on RP2040_CLK_GPOUT0
+
+	config RP2040_CLK_GPOUT0_SRC_REF
+		bool "REF (6 - 12 MHz)"
+		---help---
+			Reference clock that is always running unless in DORMANT mode. Runs from
+			Ring Oscillator (ROSC) at power-up but can be switched to Crystal
+			Oscillator (XOSC) for more accuracy.
+	config RP2040_CLK_GPOUT0_SRC_SYS
+		bool "SYS (125 MHz)"
+		---help---
+			System clock that is always running unless in DORMANT mode. Runs from
+			clk_ref at power-up but is typically switched to a PLL.
+	config RP2040_CLK_GPOUT0_SRC_USB
+		bool "USB (48 MHz)"
+		---help---
+			USB reference clock. Must be 48MHz.
+	config RP2040_CLK_GPOUT0_SRC_ADC
+		bool "ADC (48 MHz)"
+		---help---
+			ADC reference clock. Must be 48MHz.
+	config RP2040_CLK_GPOUT0_SRC_RTC
+		bool "RTC (46875 Hz)"
+		---help---
+			RTC reference clock. The RTC divides this clock to generate a 1 second reference.
+endchoice
+
+config RP2040_CLK_GPOUT0_DIVINT
+	int "Divisor (Integer)"
+	depends on RP2040_CLK_GPOUT0
```

Comment:
remove

Reply:
I'm not sure what you'd like me to remove here.  Please explain what is wrong with this section.

Reply:
I mean removing "depends on RP2040_CLK_GPOUT0" since "if RP2040_CLK_GPOUT0" at line 681 already do the same thing.

Reply:
Thanks.  I've made some changes that I believe fix these issues.